tilings/strategies/verification.py

```diff
@@ -280,24 +343,62 @@ def formal_step(self) -> str:
         return "tiling is a subclass of the original tiling"
 
     def get_terms(self, comb_class: Tiling, n: int) -> Terms:
-        raise NotImplementedError(
-            "Not implemented method to count objects for one by one verified tilings"
-        )
+        terms = super().get_terms(comb_class, n)
+        if comb_class.requirements:
+            if len(comb_class.requirements) == 1:
+                comp_spec = self.get_complement_spec(comb_class.remove_assumptions())
+            else:
+                raise NotImplementedError(
+                    "Not implemented counting for one by one with two or more reqs"
+                )
+            comp_terms = comp_spec.get_terms(n)
+            terms = Counter({a: b - comp_terms[a] for a, b in terms.items()})
```

Comment:
Trying to understand this part. Going into this line, won't terms just be a dict with a single entry, key is tuple() and value is a number?

Reply:
With the latest two commits this is now true. I'll simplify this line.